pages/ksphere/konvoy/latest/networking/custom-domain/index.md

```diff
@@ -29,10 +29,10 @@ In AWS, the `external-dns` can be configured as:
         accessKey:  <aws access key>
       region: <aws region>
       preferCNAME: true
-      policy: sync
-      txtPrefix: local-
-      domainFilters:
-      - <custom domain>
+    policy: sync
```

Comment:
This file needs header info at the top otherwise it doesn't actually appear on the site. Something like https://github.com/mesosphere/dcos-docs-site/pull/2784/files#diff-d428a6a06bcfef27c0a516985dd32346R4-R7

```
title: Custom Domain
menuWeight: 2
excerpt: Configure Konvoy to use a custom domain
enterprise: false
---	
```

Reply:
actually you can see it in the current docs when you try to click the drop down of `Networking` on the left navbar https://docs.d2iq.com/ksphere/konvoy/latest/networking/

Reply:
I fixed this in a commit i did.